network-api/networkapi/wagtailpages/templates/wagtailpages/research_library_page.html

```diff
@@ -52,12 +52,19 @@ <h1>{{ page.title }}</h1>
         </div>
         <div class="tw-h4-heading tw-mb-0">
           {# RESULTS COUNT #}
-          {% blocktranslate count counter=research_detail_pages_count trimmed %}
-            <strong>{{ research_detail_pages_count }}</strong> result
-          {% plural %}
-            <strong>{{ research_detail_pages_count }}</strong> results
-          {% endblocktranslate %}
-          {% if search_query %}{% translate 'for' context 'Heading describing the count of search results for a given search term' %} <q>{{ search_query }}</q>{% endif %}
+          {% if search_query %}
+            {% blocktranslate count counter=research_detail_pages_count trimmed %}
+              <strong>{{ research_detail_pages_count }}</strong> result for <q>{{ search_query }}</q>
+            {% plural %}
+              <strong>{{ research_detail_pages_count }}</strong> results for <q>{{ search_query }}</q>
+            {% endblocktranslate %}
+          {% else %}
+            {% blocktranslate count counter=research_detail_pages_count trimmed %}
+              <strong>{{ research_detail_pages_count }}</strong> result
+            {% plural %}
+              <strong>{{ research_detail_pages_count }}</strong> results
+            {% endblocktranslate %}
+          {% endif %}
```

Comment:
The duplication is gruesome 😅

Reply:
Yeah… I see how it can hurt a dev eyes 😅